content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -30,6 +30,20 @@ Specifically, Odoo's predictive lead scoring uses the *naive Bayes* probability
    P(A | B) = \frac{P(A) \times P(B | A)}{P(B)}
    \end{equation}
 
+Breaking down the equation:
+
+- P(A|B) = The probability of a successful lead *in this case*
+- P(A) = The overall probability of a lead being successful regardless of the conditions
+- P(B|A) = The probability of this being the case given a lead is successful
+- P(B) = The probability of this being the case
+
+The term *in this case* refers to the variables that can affect a lead being successful in Odoo.
```

Comment:
Italicize "case" throughout the equation breakdown to make it it clear this definition applies wherever "case" is used, not just "in this case"